src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1595341076.inc.php

```diff
@@ -0,0 +1,18 @@
+<h1>Build #1595341076</h1>
+<h2>Date: 2020-07-21</h2>
+<div class="changelog">
+    - #478: Hide "View in category window" as it does not help with the sidebar (and is now replaced)
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'modifier' => 'hidden',
```

Comment:
add deprecation info in help text

Reply:
There is already a deprecation note from 6.3.0 there.